scripts/start-setupMounts

```diff
@@ -24,8 +25,8 @@ else
 fi
 
 if [ -d /plugins ]; then
-  case ${TYPE} in
-    SPIGOT|BUKKIT|PAPER|MAGMA)
+  case ${FAMILY} in
+    SPIGOT|HYBRID)
```

Comment:
excuse my ignorance, but where is HYBRID coming from?

Reply:
It's a convention I had to make up for grouping all the various server types. Here's one example

https://github.com/itzg/docker-minecraft-server/blob/f29f098b2628666c50a2cb3448eecb61614fef6d/scripts/start-deployMagma#L92

Reply:
I see it now. This looks good to me if you've tested it. 👍

Reply:
Thanks! I'll go ahead and merge it since I had already forgotten about this PR 😊